src/components/Navigation/NavigationItemTrigger/NavigationItemTrigger.tsx

```diff
@@ -51,6 +51,14 @@ const NavigationItemTrigger = forwardRef<HTMLLIElement, NavigationItemTriggerPro
     const navListElement = findChildrenOfType<NavigationListProps>(children, NavigationList);
     const { closeMenu } = React.useContext(HeaderContext);
 
+    React.useEffect(() => {
+      if (isSubmenuOpened && navListElement && navListElement[0]?.props?.id) {
+        focusElementById(navListElement[0].props.id, true);
+      } else if (!isSubmenuOpened && navListElement && navListElement[0]?.props?.id) {
+        focusElementById(navListElement[0].props.id, false);
```

Comment:
this actually needs to put focus back on the NavigationItem trigger when blurred